src/verifiers/documentStatus/didSigned/didSignedDocumentStatus.ts

```diff
@@ -143,31 +171,47 @@ const verifyV3 = async (document: SignedWrappedDocument<v3.OpenAttestationDocume
   }
 
   const issuedOnAll = verificationResult.verified;
-  const revokedOnAny = !(metaData.proof.revocation?.type === "NONE");
+
+  const revocationStatus =
+    metaData.proof.revocation.type === v3.RevocationType.RevocationStore
+      ? await isRevokedOnDocumentStore({
+          documentStore: metaData.proof.revocation?.address,
```

Comment:
that's weird to use optional param here and not above (metaData.proof.revocation.type)

Reply:
`metaData.proof.revocation.type` for v3 is not optional though
its inconsistent with v2, for v2, `issuers[x].revocation.type` is optional.

Reply:
I mean
- `metaData.proof.revocation.type` L176
- `metaData.proof.revocation?.address` L178

same object (`metaData.proof.revocation`), but once used with optional and not the other

Reply:
yea sorry am not reading correctly